Help/MAC_Hints.rst

```diff
@@ -13,7 +13,15 @@ Coverage Installation Hints for macOS Users:
     $ brew install python3
     $ python3 --version   # To check the installed version
 
-3. Create Virtual Environments with pyvenv
+3. Install pyenv with homebrew
```

Comment:
hmm, it should be explained in the main setup page... maybe a note @Nosferatul if you feel it is necessary?